Doppler.Sap/Mappers/BusinessPartner/BusinessPartnerForArMapper.cs

```diff
@@ -4,14 +4,59 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Net.Mail;
-using System.Threading.Tasks;
 
 namespace Doppler.Sap.Mappers.BusinessPartner
 {
-    public class BusinessPartnerForArMapper : IBusinessPartnerMapper
+    public class BusinessPartnerForArMapper : BusinessPartnerMapper, IBusinessPartnerMapper
```

Comment:
In general, composition is simpler and flexible than heritage.

Reply:
I will review if composition can be implemented in this case.

Reply:
I talked with Andres and for now is ok but I will create a new task to implement composition.